tests/net_hosted/ssl_verify_callback.py

```diff
@@ -0,0 +1,37 @@
+# test ssl verify_callback
+
+import io
+import socket
+import tls
+
+
+def verify_callback(cert, depth):
+    print("verify_callback: ", type(cert), len(cert) > 100, depth)
```

Comment:
This doesn't need a space after the colon (Python will add spaces between printed arguments).

Similarly with `verify_callback_fail` below.